syntax = "proto3";

package lesson_plan;

service L2_tools {
  rpc GenerateLessonPlan (LessonPlanRequest) returns (stream LessonPlanResponse) {}
}

message LessonPlanRequest {
  string title = 1;
  string description = 3;
  string file_url = 2;
  string teacher_note = 4;
  string format = 5;
  uint64 teacher_id = 6;
  uint64  course_id = 7;
}

message LessonPlanResponse{
  string content = 1;
}